src/amp-story-player/amp-story-player-impl.js

```diff
@@ -559,7 +559,7 @@ export class AmpStoryPlayer {
       setStyle(iframeEl, 'backgroundImage', story.posterImage);
     }
     iframeEl.classList.add('story-player-iframe');
-    iframeEl.setAttribute('allow', 'autoplay');
+    iframeEl.setAttribute('allow', 'autoplay; web-share');
```

Comment:
could we add a simple test for this

Reply:
Unfortunately, it would be hard to mimic a native(e.g. Android, iOS) environment to test it, even though the change here is simple. Good thing is that I have tested manually on my phone  : )

Reply:
got it. LGTM